playbooks/common/openshift-cluster/upgrades/rpm_upgrade.yml

```diff
@@ -7,4 +7,4 @@
   when: "{{ not openshift.common.is_atomic | bool }}"
```

Comment:
@dgoodwin @tbielawa @ewolinetz Since this seems to be re-used now, PyYAML should probably be installed outside of this task, otherwise we are trying to install it for each package through the upgrade.